packages/trader/src/AppV2/Hooks/useIsVirtualKeyboardOpen.ts

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+type TWindow = Window & { height?: number; scale?: number };
+
+const useIsVirtualKeyboardOpen = (target_id: string) => {
+    const [is_focus, setIsFocus] = React.useState(false);
+    const [is_open, setIsOpen] = React.useState(false);
+
+    const RATIO = 0.75;
```

Comment:
What is this number?

Reply:
Let's imagine, that full height is 100 (window.screen.height) and the available space on the screen is 90 (target.height * target.scale). Then 90 / 100 = 0.9.
In case if virtual keyboard appeared the available space will be smaller, e.g 70, then 70/100 = 0.7. 
So, 0.75 is a ratio of available space to all space with corresponds with opened virtual keyboard.
I'll leave a comment in the code